src/Headings/HeadingOne.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+const Heading = styled.h1`
+  font-family: ${({ theme }): string => theme.font.heading.family};
+  font-size: ${({ theme }): string => theme.font.heading.size};
+  font-weight: ${({ theme }): string => theme.font.heading.weight};
+  color: ${({ theme }): string => theme.font.heading.color};
+`;
+
+const HeadingOne: FunctionComponent = (props): ReactElement => {
```

Comment:
`FunctionComponent` should take an argument describing the props for the component (see other components in the project for examples).

Reply:
Does that still apply even if there are no props (other than `children`?

Reply:
Fixed in 76e66fd

Reply:
I see this is resolved, but just to clarify: We should still explicitly specify that `children` is the only prop expected, as a signal (for `tsserver`, docs, etc,) that there are no other things that can be added.